### PR TITLE
changed in db-lb for multicloud scenario

### PR DESCRIPTION
**Pull request by arikyakir**
changed naes for multicloud scenario

### CODE REVIEW

File: db-lb-app/infrastructure/openstack.yaml [hunk 1]
```diff
@@ -8,24 +8,33 @@ imports:
...
 
-  region_name:
-    type: string
+  openstack_region:
+    description: ""
+    type:
+    default: { get_secret: openstack_region }
 
   public_subnet_cidr:
+    description: ""
```
Comment (EarthmanT) [inline db-lb-app/infrastructure/openstack.yaml]: Why do you need empty descriptions? Remove them if they aren't going to say anything. They are not required.

File: db-lb-app/infrastructure/openstack.yaml [hunk 2]
```diff
@@ -8,24 +8,33 @@ imports:
 
 inputs:
 
-  region_name:
-    type: string
+  openstack_region:
+    description: ""
+    type:
+    default: { get_secret: openstack_region }
```
Comment (EarthmanT) [inline db-lb-app/infrastructure/openstack.yaml]: Maybe we should remove this and just change `{ get_input: region_name }` below to `{get_secret: openstack_region }`?

File: db-lb-app/infrastructure/openstack.yaml [hunk 3]
```diff
@@ -8,24 +8,33 @@ imports:
...
+    default: { get_secret: base_image_id }
 
   flavor_id:
-    type: string
+    description: ""
+    type:
+    default: { get_secret: base_flavor_id }
 
   resource_name_prefix:
+    description: "prefix to use for this elemts (VM) e.g db,lb"
```
Comment (EarthmanT) [inline db-lb-app/infrastructure/openstack.yaml]: Please delete all the descriptions.

File: db-lb-app/infrastructure/openstack.yaml [hunk 4]
```diff
@@ -135,15 +144,15 @@ node_templates:
       client_config: *client_config
       use_external_resource: true
       resource_config:
-        id: { get_capability: [ openstack, public_network ] }
+        id: { get_capability: [ openstack-network, public_network ] }
```
Comment (EarthmanT) [inline db-lb-app/infrastructure/openstack.yaml]: Please don't change the name of this, if you want it to be differnt in the demo add and input `network_deployment_name` and use here { get_capapability: [ { get_input: network_deployment_name }, public_network ] }`

File: db-lb-app/infrastructure/openstack.yaml (hunk 2 above)
Comment (arikyakir) [reply db-lb-app/infrastructure/openstack.yaml]: this way the user can override if he dont want the secret. more flexible

File: db-lb-app/infrastructure/openstack.yaml (hunk 4 above)
Comment (arikyakir) [reply db-lb-app/infrastructure/openstack.yaml]: done

File: db-lb-app/infrastructure/openstack.yaml (hunk 1 above)
Comment (arikyakir) [reply db-lb-app/infrastructure/openstack.yaml]: added description content

**Pull request merged**